contracts/vault/IVaultSponsoring.sol

```diff
@@ -24,10 +24,15 @@ interface IVaultSponsoring {
      * there are no claimers and the yield generated is donated to the vault.
      * The amount is locked until the timestamp specified in @param _lockedUntil.
      *
+     * @param _inputToken The input token to deposit.
      * @param _amount Amount to sponsor.
      * @param _lockedUntil When the sponsor can unsponsor the amount.
      */
-    function sponsor(uint256 _amount, uint256 _lockedUntil) external;
+    function sponsor(
+        address _inputToken,
```

Comment:
could we maybe only allow sponsors on the underlying currency?

Reply:
why?

Reply:
just wondering if it helps remove complexity

Reply:
doesn't change much (just this extra argument on the sponsor function). and in my opinion would make the API look inconsistent